source/includes/_dataset_security.md

```diff
@@ -0,0 +1,339 @@
+# Dataset security
+
+The dataset security is defined through 3 variables:
+
+* a general access policy
+* specific rulesets for users and groups
+* a default ruleset that applies to anybody else
+
+The general access policy sets whether the dataset is accessible to anyone or just a few specific users.
+
+### Restricted datasets
+
+If the dataset is [set as restricted](#set-the-general-access-policy), then the dataset will only appear in the catalog for users who have a ruleset declared for them, either directly or through a group. Other users won't have any access to the dataset.
+
+As a result, the default security ruleset has no meaning for restricted datasets.
+
+### Standard access datasets
+
+All people having access to the domain will be able to see the dataset in the catalog.
+
+If a user has at least a ruleset declared for them (directly, through a group or both), they will be able to see the union of everything each of the rulesets grants access to.
+
+Otherwise, if nothing has been declared for them (neither directly nor through a group), they will be able to see what the default ruleset grants access to.
```

Comment:
Same idea about starting a paragraph with a conjunction. With Otherwise, you're not starting a new idea, but rather expanding on an existing one.

Second point: I think "nothing" is a bit too vague, "no ruleset" would be better. Granted it is a bit repetitive, but technical writing is about precision.